db/seeds.rb

```diff
@@ -8,3 +8,4 @@
 require File.dirname(__FILE__) + '/seed_helper.rb'
 
 create_roles_and_permissions
+populate_languages
```

Comment:
I've added a hook to deployment to run the method to populate the languages table from the CSV. I think this should be ok, but I don't have any method for testing it.